SafeGraph_py_functions/SafeGraph_py_functions.py

```diff
@@ -113,65 +113,99 @@ def unpack_json_separate(df_, json_column='visitor_home_cbgs', key_col_name='vis
     return output
 
 
-def unpack_json_together(df, json_column='visitor_home_cbgs', key_col_name='visitor_home_cbg',
+def unpack_json_and_merge(df, json_column='visitor_home_cbgs', key_col_name='visitor_home_cbg',
                          value_col_name='cbg_visitor_count', keep_index=False):
     if (keep_index):
         df['index_original'] = df.index
     df = df.dropna(subset=[json_column]).copy()  # Drop nan jsons
     df.reset_index(drop=True, inplace=True)  # Every row must have a unique index
-    df_exp = unpack_json_separate(df, json_column=json_column, key_col_name=key_col_name, value_col_name=value_col_name)
+    df_exp = unpack_json(df, json_column=json_column, key_col_name=key_col_name, value_col_name=value_col_name)
     df = df.merge(df_exp, left_index=True, right_index=True).reset_index(drop=True)
     return df
 
+def explode_json_array(df_, array_column = 'visits_by_day', new_col='day_visit_counts',place_key='safegraph_place_id', file_key='date_range_start', array_sequence='day', keep_index=False, verbose=True, zero_index=False):
+    # This function vertically explodes an array column in SafeGraph data and creates a second new column indicating the index value from the array
```

Comment:
@Trippl7777  like your other functions, let's move this commentary up to the help section?

Reply:
will do